### PR TITLE
Fix typos, wrong translated texts (tr.json)

### PR DESCRIPTION
**Pull request by BurakYs**
I have fixed some typos, fixed unnecessary uppercase and lowercase letters and fixed wrong translated texts

### CODE REVIEW

**Comment by BurakYs** [comment]
I'm Burak#2466 on discord, i forgot to add my discord username to description

File: tr.json [hunk 1]
```diff
@@ -299,50 +299,50 @@
...
             },
             "oauth" : {
-                "title" : "fnbr.co ya giriş yap",
-                "discord-button" : "Discord ile giriş Yap",
+                "title" : "fnbr.co'ya Giriş Yap",
+                "discord-button" : "Discord ile Giriş Yap",
                 "legacy-text" : "Alternatif olarak, 2018'den kalma hesabınız varsa:",
                 "twitter-button" : "Twitter",
-                "privacy-text" : "Giriş yaparak, hesabımızda belirtilmiş olan ve hesabımızda belirtilen verileri kaydetmemizi kabul edersiniz [Gizlilik Politikası](/privacy)."
+                "privacy-text" : "Giriş yaparak hesabınızla ilgili verileri [gizlilik politikamızda] belirtildiği şekilde saklamamızı kabul etmiş olursunuz."
```
Comment (Krymonota) [inline tr.json:310]:
```suggestion
                "privacy-text" : "Giriş yaparak hesabınızla ilgili verileri [gizlilik politikamızda](/privacy) belirtildiği şekilde saklamamızı kabul etmiş olursunuz."
```

**Review by Krymonota — CHANGES_REQUESTED** [review]
Hi. Thank you for your contribution!

There's a missing link that you need to fix, but you can do so by simply applying my suggestion. Other than that, the changes look good to me and I'd be happy to merge them after you've fixed the mentioned problem 🙂

**Review by Krymonota — APPROVED** [review]
Thanks, looks good now 👍

**Pull request merged**